assignment5/unknownLanguage_test.py

```diff
@@ -0,0 +1,39 @@
+"""Tests for unknownLanguage."""
+
+from unknownLanguage import unknownLanguage
+import unittest
+
+class UnknownLanguageTest(unittest.TestCase):
+    def testEqualLenghtCases(self):
+        self.assertEqual(unknownLanguage(["TAR", "ART", "RAT", "CAT", "CAR"]),
+                         ['T', 'A', 'R', 'C'])
+        self.assertIn(unknownLanguage(["ART", "RAT", "CAT", "CAR"]),
+                      [['A', 'T', 'R', 'C'],
+                       ['T', 'A', 'R', 'C']])
+        self.assertIn(unknownLanguage(["QHS", "QST", "OOQ", "OTS",
+                                       "NTO", "NTS", "SNS", "TNS"]),
+                      [['Q', 'H', 'O', 'N', 'S', 'T'],
+                       ['Q', 'H', 'O', 'S', 'N', 'T'],
+                       ['H', 'Q', 'O', 'N', 'S', 'T'],
+                       ['H', 'Q', 'O', 'S', 'N', 'T']])
+
+    def testDifferentLengthCase(self):
+        self.assertEqual(unknownLanguage(["a", "ab", "ac", "bca", "bcad",
+                                          "bcd", "cdba", "cdc", "daac"]),
+                         ['a', 'b', 'c', 'd'])
+        self.assertIn(unknownLanguage(["a", "dad", "ba", "bad", "bac", ""]),
+                      [['a', 'd', 'c', 'b'],
```

Comment:
Is this because there are multiple valid alphabets?

Other alternative would be to check what you want to be true in this case.

For example:
```python
  self.assertTrue(alphabet.index('a') < alphabet.index('d')
```